### PR TITLE
Introduce printHelp() function

### LINKED ISSUE #25: Parameter --help

### PR DESCRIPTION
**Pull request by yhabteab**
resolves #25

### CODE REVIEW

File: sort.php [hunk 1]
```diff
@@ -1,5 +1,25 @@
 <?php
 
+function printHelp()
+{
+    echo "Usage: php sort.php [OPTION] < [FILE]
```
Comment (Al2Klimov) [inline sort.php]: Please use so called heredocs for such strings.

File: sort.php [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+    or: php sort.php [OPTION] files\n
```
Comment (Al2Klimov) [inline sort.php]: "file**s**"? Does the program take multiple files?

File: sort.php [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+    Without a given file name will be use the standard console output.\n
+    Sort options:\n
+    -b,                     Ignore leading blanks
```
Comment (Al2Klimov) [inline sort.php]: Why the ","?

File: sort.php [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+    -f, ignore case         Treat lowercase as uppercase
+    -n, numeric sort        Sort by numeric value
+    -R, random sort         Sort the keys using a random hash
+    -r, reverse             Reverse the result of sorting
+    -o, output=FILE         Write result in FILE instead of standard output
+    -u, unique              Only output the first of several matches
+        --merge,            Use merge sort Algorithm
+        --qsort,            Use quick sort Algorithm
```
Comment (Al2Klimov) [inline sort.php]: Why the ","s?

File: sort.php [hunk 5]
```diff
@@ -1,5 +1,25 @@
...
+    or: php sort.php [OPTION] files\n
+    Without a given file name will be use the standard console output.\n
+    Sort options:\n
+    -b,                     Ignore leading blanks
+    -f, ignore case         Treat lowercase as uppercase
+    -n, numeric sort        Sort by numeric value
+    -R, random sort         Sort the keys using a random hash
+    -r, reverse             Reverse the result of sorting
+    -o, output=FILE         Write result in FILE instead of standard output
+    -u, unique              Only output the first of several matches
```
Comment (Al2Klimov) [inline sort.php]: Why e.g. the ", ignore case"?

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: sort.php (hunk 1 above)
Comment (Al2Klimov) [reply sort.php]: Still missing.

Comment (yhabteab) [reply sort.php]: check the second commit

Comment (Al2Klimov) [reply sort.php]: "Usage" was right. I asked you to use heredocs.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [ ] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [ ] `git commit --amend --no-edit`
* [ ] `git push -f origin feature/sort-#25`

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**